app/src/helpers/helpers.test.ts

```diff
@@ -261,3 +262,27 @@ describe('removeUserAgentSpecifics', () => {
     );
   });
 });
+
+describe('getPlainTextWithoutFormatting', () => {
+  test('removes extra spaces from text', () => {
+    expect(getPlainTextWithoutFormatting('  this is a  test  ')).toBe(
+      'this is a test',
+    );
+  });
+  test('removes newlines from text', () => {
+    expect(
+      getPlainTextWithoutFormatting(`
+    this is a
+    test
+    `),
+    ).toBe('this is a test');
+  });
+  test('removes newlines and from text', () => {
+    expect(
+      getPlainTextWithoutFormatting(`
+        this is a  
+    test  
+    `),
+    ).toBe('this is a test');
+  });
```

Comment:
But I'm not sure I find these two tests to be expected behavior for such a feature. I just tried in LibreOffice, the "Paste unformatted text" feature _does_ preserve newlines.